lib/Mojo/IOLoop/Subprocess.pm

```diff
@@ -245,6 +257,24 @@ the second callback in the parent process with the results. The return values of
 the first callback and exceptions thrown by it, will be serialized with
 L<Storable>, so they can be shared between processes.
 
+=head2 run_p
+
+  my $promise = $subprocess->run_p(sub {...});
+
+Same as L</"run">, but returns a L<Mojo::Promise> object instead of accepting a
+second callback.
+
+  $subprocess->run_p(sub {
```

Comment:
This could use a comment describing the example.

Reply:
Do you have an example of what kind of description it could have? I took inspiration from https://mojolicious.org/perldoc/Mojo/UserAgent#get_p and didn't think this was any different.

Reply:
Fair enough.